src/config/config.ts

```diff
@@ -206,7 +206,7 @@ const config = convict({
           doc: "Whether the queues are deprecated",
           env: "FF_DEPRECATE_SITE_QUEUES",
           format: "required-boolean",
-          default: false,
+          default: null,
```

Comment:
I think it's okay to keep false as the default, though we should still add it to our env vars, but i'm fine either way!